sapi/cli/php_cli_server.c

```diff
@@ -356,10 +356,10 @@ static void append_essential_headers(smart_str* buffer, php_cli_server_client *c
 	}
 
 	if (!gettimeofday(&tv, NULL)) {
-		zend_string *dt = php_format_date("r", 1, tv.tv_sec, 1);
+		zend_string *dt = php_format_date("D, d M Y H:i:s", 14, tv.tv_sec, 0);
```

Comment:
Should use `sizeof("D, d M Y H:i:s") - 1` for the length.

Reply:
Fixed.